boot/boot_images.cmake

```diff
@@ -59,6 +61,53 @@ ${_efisys_file} 1
 set(ISO_MANUFACTURER "ReactOS Foundation")  # For both the publisher and the preparer
 set(ISO_VOLNAME      "ReactOS")             # For both the Volume ID and the Volume set ID
 
+# ISO image options
+set(ISO_COMMON_OPTIONS
+    -iso-level 4 -publisher ${ISO_MANUFACTURER} -preparer ${ISO_MANUFACTURER} -volid ${ISO_VOLNAME} -volset ${ISO_VOLNAME})
+
+set(ISO_LAYOUT_OPTIONS
+    -duplicates-once -no-cache-inodes -graft-points)
+
+## "El Torito" ISO boot options.
+#
+set(ISO_BOOT_OPTIONS)
+set(ISO_BOOT_REGTEST_OPTIONS) # Useful only for BootCDRegTest
+
+# BIOS-based PC boot entry (x86/x64 only)
+if(ARCH STREQUAL "i386" OR ARCH STREQUAL "amd64")
+    set(ISO_BOOT_OPTIONS
+        -eltorito-platform x86 -eltorito-boot loader/isoboot.bin -no-emul-boot -boot-load-size 4)
+    set(ISO_BOOT_REGTEST_OPTIONS
+        -eltorito-platform x86 -eltorito-boot loader/isobtrt.bin -no-emul-boot -boot-load-size 4)
+endif()
+
+# EFI boot entry
+list(APPEND EFI_SUPPORTED_ARCHS "i386" "amd64" "ia64" "arm" "arm64")
```

Comment:
Why `list(APPEND`?
Why not just set it?

Reply:
I copied the technique from https://git.reactos.org/?p=reactos.git;a=blob;f=sdk/cmake/CMakeMacros.cmake;hb=64290dd2c98f430c23a47100f655ce7e6849c3ee#l575
that seems to avoid the syntax with the explicit semicolons between the elements.